data/yaml/mspass.yaml

```diff
@@ -34,6 +34,10 @@ Database:
         type: ObjectID
         concept: ObjectId used to define a data object
         constraint: required
+      data_tag:
+        type: string
+        concept:  Used to label data at a particular stage of processing. Should be unique for each save operation.
+        constraint: normal
```

Comment:
As I mentioned in #158, this is not necessary to be in the schema. We should just implement the data_tag as part of the API.